### ISSUE TITLE
Roomba s9+

### ISSUE THREAD

**Issue by migabc** [esteban-mallen/homebridge-roomba-stv#67]
Hi, is this plugin compatible with Roomba s9+?

**Comment by Pblondin123** [comment]
yes i have a s9 and tested successfully.   See other issue 66 on issues/restrictions being investigated.
